### PR TITLE
Milestone 1: Blog app - integration specs for views

### PR DESCRIPTION
**Pull request by Gardimy**
In this project, my coding partner and I created integration tests for all of the views used in my project.

Screen before
screen1

screen after
screen2

### CODE REVIEW

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️
- Readme file professional ✔️
- All tests passed ✔️ 

## Required Changes ♻️
_Check the comments under the review._
- [ ] I noticed you did not add the screenshots as required, I will encourage you to add it so that it can be verified though I could see you bettered the query with the use of includes but this is also required. 
image

- [ ] While I can see that all your tests are passing as we both saw on the call, I will encourage you to write all the test as required as most of the tests are yet to be written, for those that you wrote which are 5, I could see you did not separate the tests, kindly separate the test also so that the test can be 29.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/views/posts/index.html.erb_spec.rb [hunk 1]
```diff
@@ -1,23 +1,16 @@
...
+    post = user.posts.create(title: 'Example Post', text: 'Post Text')
+    visit user_posts_path(user)
 
-    expect(current_path).to eq(post_path(Post.first))
+    expect(page).to have_content(user.name)
+    expect(page).to have_content(post.title)
+    expect(page).to have_content("Number of Posts: #{user.posts.count}")
+    expect(page).to have_content(post.text)
+    expect(page).to have_content("#{post.comments.count} Comments")
+    expect(page).to have_content("#{post.likes.count} Likes")
```
Comment (Lucas-Masaba) [inline spec/views/posts/index.html.erb_spec.rb:15]: - [ ] You are missing a test that asserts you can see a user's profile picture here.
image

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @Gardimy && @waelelsafty07  👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
almost-there

 ## Highlights 🍾 
- Linter checks and written tests passing :white_check_mark: 
- Made changes as requested by the previous reviewer :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gambit142 — APPROVED** [review]
Hi @Gardimy @waelelsafty07 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- Setup linters correctly ✔️ 
- Use of Rsec testing library ✔️ 
- Wrote feature spec tests ✔️

### Optional Suggestion

- [ ] The screenshots you attached don't show the log for the `Posts#Index` page but the logs for the `Users#Index` page and `root` page. Remember the `N + 1` query was mostly solved in the `index` action of the `Posts` controller. It would be nice to adjust this. 👍 😎 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**